provision/pull-images.sh

```diff
@@ -51,7 +60,8 @@ if [ -z "${NAME_PREFIX}" ]; then
         quay.io/cilium/cilium-runtime:2018-10-29 \
         quay.io/coreos/etcd:v3.2.17 \
         quay.io/coreos/etcd:v3.3.9; \
```

Comment:
@ianvernon This `;` is at the wrong place

Reply:
Good catch.

Reply:
Fixed.